.travis.yml

```diff
@@ -1,15 +1,14 @@
-language: ruby
-rvm:
-  - 2.3.0
 sudo: false
 cache: bundler
+language: ruby
 env:
-  matrix:
-    - SOLIDUS_BRANCH=master
-    - SOLIDUS_BRANCH=v1.3
-    - SOLIDUS_BRANCH=v1.2
-    - SOLIDUS_BRANCH=v1.1
-before_script:
-  - bundle exec rake test_app
-script:
-  - bundle exec rspec spec
+  - SOLIDUS_BRANCH=v1.1   DB=mysql
+  - SOLIDUS_BRANCH=v1.2   DB=mysql
+  - SOLIDUS_BRANCH=v1.3   DB=mysql
+  - SOLIDUS_BRANCH=master DB=mysql
+  - SOLIDUS_BRANCH=v1.1   DB=postgres
+  - SOLIDUS_BRANCH=v1.2   DB=postgres
+  - SOLIDUS_BRANCH=v1.3   DB=postgres
+  - SOLIDUS_BRANCH=master DB=postgres
+rvm:
+  - 2.3
```

Comment:
Interesting, I had originally put `2.3` and Travis yelled at me for not being specific.